oxford-metadata.ttl

```diff
@@ -148,28 +148,33 @@
     a :Category ;
     rdfs:label "Physical properties" .
 
-:CellMembraneCurrent
+:IonicCurrent
+    a rdfs:Class ;
```

Comment:
This will mean `IonicCurrent` doesn't show up as a category in its own right in the annotation tool. Is that reasonable @mirams @MichaelClerx ? All terms in this category should also appear in another category, and I wanted to reduce duplication in the tool.

Reply:
I think that's fine for now.